index.html

```diff
@@ -745,129 +758,172 @@ <h2 class="display-4 text-center" id="data_header">Loading data...</h2>
           </ul>
           <!-- Mage Arcane -->
           <ul class="tc-box" id="tc_mage_arcane" hidden>
+            <li><span class="translate_arcane">Arcane</span> Guides
+              <a href="https://www.icy-veins.com/wow/arcane-mage-pve-dps-guide" target="blank">Icy Veins</a>&nbsp;/&nbsp;
```

Comment:
Please add altered-time to the guide links. Format should look like this:

<Spec> Guides: Altered-Time / Icy Veins / Wowhead

so...sorted by name